### ISSUE TITLE
using sum() when there are commas in table cells?

### ISSUE THREAD

**Issue by luckman212** [tgrosinger/advanced-tables-obsidian#76]
This may be a hard limitation but, I was wondering, is it somehow possible to support math e.g. `sum()` functions when the numbers contain commas? E.g. see below:

```md
| Item              | Grams |
| ----------------- | -----:|
| Whole Wheat Flour |   110 |
| Bread Flour       |   748 |
| Warm Water        | 1,691 |
| Salt              |    18 |
| Starter           |    40 |
| **Total Grams**   |   917 |
<!-- TBLFM: @>$2=sum(@I..@-1) -->
```

result: 
![image](https://user-images.githubusercontent.com/1992842/113516832-78b24680-954a-11eb-89a3-6adf5b64517a.png)

(should be 2607)

**Comment by tgrosinger** [comment]
Yeah, should be able to do that. I'll take a look next time I have that code open.

**Comment by tgrosinger** [comment]
Ah, ran into a hitch here. I can't blindly ignore commas because in some locales they are used instead of decimal points. That makes this a bit trickier. It looks like there is a native way to format numbers to the current locale, but not parse them. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

**Comment by luckman212** [comment]
Hmm, I see. What about adding a setting box where the user can input a "character to ignore" (simpler than trying to work this out automatically based on the num format or their regional settings etc).

I haven't looked yet at the code but if the function just runs a `str.replace(/char/g,"")` to strip them before performing the math—could that work?
